packages/teams/src/utils/get-user-creatable-teams.ts

```diff
@@ -4,27 +4,38 @@ import { WELLKNOWNTEAMS } from "../well-known-teams";
 import { PRE_9_1_WELLKNOWNTEAMS } from "../pre-9-1-well-known-teams";
 import { canUserCreateTeamInProduct } from "./can-user-create-team-in-product";
 import { IGroupTemplate } from "../types";
+import { filterUserPrivsBySubscriptionType } from "./filter-user-privs-by-subscription-type";
+import { updateTemplateBasedOnPrivs } from "./update-template-based-on-privs";
 
 /**
  * Return array of group templates that the current user has licensing
  * and privs to create in the current environment (AGO vs Portal)
  * @param {object} user
  * @param {string} environment
+ * @param {string} portalApiVersion
+ * @param {string} subscriptionInfoType
  */
 export function getUserCreatableTeams(
   user: IUser,
   environment: HubProduct,
-  portalApiVersion: string
+  portalApiVersion: string,
+  subscriptionInfoType: string
 ): IGroupTemplate[] {
   // TODO: remove this when needed (after may 1st 2021)
```

Comment:
Since we are after May 1, can we do the removal? i.e. nix the PRE_9_1_WELLKNOWNTEAMS file, and this block of logic?

Reply:
Will be done in it's own PR.